src/outlines.js

```diff
@@ -38,15 +38,15 @@ const layout = exports._layout = (config = {}, points = {}, units = {}) => {
     const parsed_glue = u.deepcopy(a.sane(config, 'outlines.glue', 'object')())
     for (let [gkey, gval] of Object.entries(parsed_glue)) {
         a.unexpected(gval, `outlines.glue.${gkey}`, ['top', 'bottom', 'waypoints', 'extra'])
-    
+
```

Comment:
Looks like my editor tried to save some bytes, I'll revert these unnecessary changes before merge